From cd3e2d41a986904304f1ad8096574e0ec0262fde Mon Sep 17 00:00:00 2001
From: Nizar Haouati <nizar.haouati@intel.com>
Date: Wed, 24 Apr 2013 09:24:57 +0300
Subject: [PORT FROM MAIN] Telephony: Fix typo

BZ: 71115

This patch fixes the typo in CatService.

Change-Id: Icbfbe40a0a4ae2cd4ecdb23de4b0f95886430dbe
Orig-Change-Id: I7264ce6fda980b4bf209acfb49f2c036983a8a45
Signed-off-by: Nizar Haouati <nizar.haouati@intel.com>
Signed-off-by: Arun Ravindran <arun.ravindran@intel.com>
---
 .../android/internal/telephony/cat/CatService.java |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/java/com/android/internal/telephony/cat/CatService.java b/src/java/com/android/internal/telephony/cat/CatService.java
index 329cefe..e2bd39e 100644
--- a/src/java/com/android/internal/telephony/cat/CatService.java
+++ b/src/java/com/android/internal/telephony/cat/CatService.java
@@ -97,7 +97,7 @@ public class CatService extends Handler implements AppInterface {
     private static final int DEV_ID_KEYPAD      = 0x01;
     private static final int DEV_ID_UICC        = 0x81;
     private static final int DEV_ID_TERMINAL    = 0x82;
-    static final String STK_DEFAULT = "Defualt Message";
+    static final String STK_DEFAULT = "Default Message";
 
     /* Intentionally private for singleton */
     private CatService(CommandsInterface ci, UiccCardApplication ca, IccRecords ir,
-- 
1.7.4.1

